HIP/hip-1056.md

```diff
@@ -0,0 +1,1747 @@
+Aggregated signatures allow the verification of multiple node signatures to be processed using a single public key.
+Therefore, a user can verify that a block was signed by a sufficient stake weight within the network by verifying a single
+aggregated signature. TSS-BLS signatures were selected because of their ability to support partial share signatures
+assigned on a weighted number of shares per node with a threshold value required for verification. Additionally,
+adopting TSS-BLS allows the network to utilize a single semi-permanent public ledger id to verify the signature. This
+offers a valuable system optimization as no matter how much the network node membership changes, the same ledger id can
+be used to verify the aggregated signature. TSS-BLS signatures are also reasonably efficient when used within smart
+contracts. A state proof based on this signature can be verified by a smart contract on another ledger for little cost.
+
+A subsequent HIP will be created to outline the changes and justification required for implementing Hedera TSS-BLS
+signatures.
+
+Notably the block stream will enable 3 types of proof
+1. Block Proof - This proof proves the blockchain and illustrates that given a starting block and its proof, a given
+set of transaction inputs and outputs as well as resulting state changes and a final merkle root hash was agreed upon
+by the majority stake weight of the network. 
+In detail the Block Proof is a message in the Block Stream that contains a Ledger ID signature on the root hash of the Block Merkle
+Tree, as well as all of the information necessary to, with the Block Items for that Block, reconstruct the Block Merkle
+Tree, calculate the root hash, and verify the signature with the network Ledger ID.
+2. Block Item Proof - This proof proves that a given object was present in a Block. This is just as powerful as proving
+a value was in state. In detail, the Block Item Proof is a Merkle Proof created for a single BlockItem within a Block.
+This proves that the network consensus agreed to the content of that specific item (input, output, or state change) in
+that specific block. This can prove, among other values, the content of an HCS message, the state of an Account changed
+in that block, or the content of a transaction submitted in that block. We can construct a BlockItem Proof from a Block
+Proof and contents of the Block. 
+3. State Proof - This proof proves a given state value at a given block number was agreed upon by a majority stake
+weight of the network. In detail, teh state proof is a Merkle Proof created for a value in network consensus state at a
+particular point in consensus time. This proves that the network consensus state contained a particular value at a
+specific consensus timestamp. This might prove, among other values, the balance of an Account, the state of a Smart
+Contract, or the properties of an HTS non-fungible token. 
```

Comment:
Minor typo fix:
```suggestion
3. State Proof - This proof proves a given state value at a given block number was agreed upon by a majority stake
weight of the network. In detail, the state proof is a Merkle Proof created for a value in network consensus state at a
particular point in consensus time. This proves that the network consensus state contained a particular value at a
specific consensus timestamp. This might prove, among other values, the balance of an Account, the state of a Smart
Contract, or the properties of an HTS non-fungible token. 
```

Reply:
Fixed